cli/roger_push.py

```diff
@@ -454,7 +454,8 @@ def main(self, settings, appConfig, frameworkObject, hooksObj, args):
                                                          container_name, environment, act_as_user)
                         # // operator does floor division, rounds up to integer
                         color = "green" if resp.status_code // 100 == 2 else "red"
-                        print(colored(json.dumps(resp.json(), indent=4), color))
+                        if not resp.status_code == 204: # empty response
+                            print(colored(json.dumps(resp.json(), indent=4), color))
```

Comment:
Why not just attempt parsing the JSON here and catch a `ValueError` in the case that it fails?

Reply:
Basically:
```python
try:
    # decode JSON and output here
except ValueError:
    # simply output result string here instead
```